Formula/xmrig.rb

```diff
@@ -1,8 +1,7 @@
 class Xmrig < Formula
   desc "Monero (XMR) CPU miner"
   homepage "https://github.com/xmrig/xmrig"
-  url "https://github.com/xmrig/xmrig/archive/v2.6.4.tar.gz"
-  sha256 "94640ba6d3971a6ba91113ce24a5afec95189cf007f8146011f9a3b367011a69"
```

Comment:
The checksum is required.

Reply:
Sorry for missing that. The sha256 had been added.